pkgs/by-name/wc/wcurl/package.nix

```diff
@@ -0,0 +1,43 @@
+{
+  stdenv,
+  lib,
+  curl,
+  fetchFromGitHub,
+  makeWrapper,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "wcurl";
+  version = "2024-07-07";
```

Comment:
Date versions are normally separated by a point, the [latest release](https://salsa.debian.org/debian/wcurl/-/tags/2024.07.10) uses this format

Reply:
@lelgenio thank you. I'm glad that upstream improved their versioning. I've made the changes you suggested.